dev-docs/modules/fledgeForGpt.md

```diff
@@ -2,172 +2,65 @@
-in their Open Bidding and Exchange Bidding APIs. More details on that can be found
-[here](https://github.com/google/ads-privacy/tree/master/proposals/fledge-rtb#bid-request-changes-indicating-interest-group-auction-support)
-In practice, this looks as follows:
-
-```js
-pbjs.addAdUnits({
-    code: "my-adunit-div",
-    // other config here
-    ortb2Imp: {
-        ext: {
-            ae: 1
+pbjs.setConfig({
+    paapi: {
+        gpt: {
+            autoconfig: false
         }
     }
-});
+})
 ```
 
-## Bid Adapter Integration
-
-Chrome has enabled a two-tier auction in FLEDGE. This allows multiple sellers (frequently SSPs) to act on behalf of the publisher with
-a single entity serving as the final decision maker. In their [current approach](https://github.com/google/ads-privacy/tree/master/proposals/fledge-multiple-seller-testing),
-GPT has opted to run the final auction layer while allowing other SSPs/sellers to participate as
-[Component Auctions](https://github.com/WICG/turtledove/blob/main/FLEDGE.md#21-initiating-an-on-device-auction) which feed their
-bids to the final layer. To learn more about Component Auctions, go [here](https://github.com/WICG/turtledove/blob/main/FLEDGE.md#24-scoring-bids-in-component-auctions).
-
-The FLEDGE auction, including Component Auctions, are configured via an `AuctionConfig` object that defines the parameters of the auction for a given
-seller. This module enables FLEDGE support by allowing bid adaptors to return `AuctionConfig` objects in addition to bids. If a bid adaptor returns an
-`AuctionConfig` object, Prebid.js will register it with the appropriate GPT ad slot so the bidder can participate as a Component Auction in the overall
-FLEDGE auction for that slot. More details on the GPT API can be found [here](https://developers.google.com/publisher-tag/reference#googletag.config.componentauctionconfig).
-
-Modifying a bid adapter to support FLEDGE is a straightforward process and consists of the following steps:
-
-1. Detecting when a bid request is FLEDGE eligible
-2. Responding with AuctionConfig
-
-FLEDGE eligibility is made available to bid adapters through the `bidderRequest.fledgeEnabled` field.
-The [`bidderRequest`](/dev-docs/bidder-adaptor.html#bidderrequest-parameters) object is passed to
-the [`buildRequests`](/dev-docs/bidder-adaptor.html#building-the-request) method of an adapter. Bid adapters
-who wish to participate should read this flag and pass it to their server. FLEDGE eligibility depends on a number of parameters:
-
-1. Chrome enablement
-2. Publisher participation in the [Origin Trial](https://developer.chrome.com/docs/privacy-sandbox/unified-origin-trial/#configure)
-3. Publisher Prebid.js configuration (detailed above)
-
-When a bid request is FLEDGE enabled, a bid adapter can return a tuple consisting of bids and AuctionConfig objects rather than just a list of bids:
+You may then use `setPAAPIConfigForGPT`, typically from a `bidsBackHandler`:
```

Comment:
We should add more context here to ensure pbjs.setPAAPIConfigForGPT(); function is invoked within the bidsBackHandler whenever new bids are requested, such as when refreshing adSlots. This is important because auctionConfig needs to be linked with a GPT slot manually when autoconfig is disabled.

Reply:
cf. https://github.com/prebid/Prebid.js/issues/11242